.drone.yml

```diff
@@ -13,6 +13,7 @@ pipeline:
     image: lampepfl/dotty:2017-10-20
     commands:
       - cp -R . /tmp/1/ && cd /tmp/1/
+      - ./project/scripts/sbt compile # to ensure that side projects (such as dotty doc) compile
       - ./project/scripts/sbt testAll
       - ./project/scripts/sbt ";dotty-bench/jmh:run 1 1 tests/run/arrays.scala"
```

Comment:
This was meant to make sure that `dotty-bench` compiles. Should it be part of dottyRoot so that we can get rid of this line?

Reply:
That would be a solution